css/main.css

```diff
@@ -416,7 +420,54 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
+.menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  z-index: 100;
+  background: #3c3a39;
+  padding: 30px;
+}
+
+.menu a {
+  color: #fff;
+}
+
+.cross {
+  float: right;
+  background: transparent;
+  color: #fff;
+}
+
+.fa-envelope {
+  display: none !important;
+}
+
```

Comment:
- [ ] As  a standard for [HTML/CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), it's recommended not to overuse `!important` in your code. You just added **2** more to the existing **4** of this overriding rule in your code to force specificity in your project. This could easily introduce a bug or conflicts in your code.

According to [W3Schools](https://www.w3schools.com/css/css_important.asp), 2 or 3 use cases is better to avoid unexpected outcomes.